vncviewer/cocoa.mm

```diff
@@ -88,28 +98,25 @@ int cocoa_capture_displays(Fl_Window *win)
 
   captured = true;
 
-  if ([nsw level] == CGShieldingWindowLevel())
+  if (cocoa_get_level(win) == CGShieldingWindowLevel())
```

Comment:
I don't think these are useful. This file is supposed to be able to use Cocoa directly. These indirection functions are only for non-ObjectiveC code.